src/pages/components/date-picker/code.mdx

```diff
@@ -1,80 +1,92 @@
 ---
 title: Date picker
 description: Date and time pickers allow users to select a single or a range of dates and times.
-tabs: ['Code', 'Usage', 'Style', 'Accessibility']
+tabs: ["Code", "Usage", "Style", "Accessibility"]
 ---
 
-## Default
-
-<ComponentDemo
-  knobs={{
-    DatePicker: ['light'],
-  }}
-  links={{
-    React:
-      'https://react.carbondesignsystem.com/?path=/story/datepicker--simple',
-    Angular:
-      'https://angular.carbondesignsystem.com/?path=/story/components-date-picker--simple',
-    Vue:
-      'http://vue.carbondesignsystem.com/?path=/story/components-cvdatepicker--default',
-    Vanilla: 'https://the-carbon-components.netlify.com/?nav=date-picker',
-  }}>{`<DatePicker dateFormat="m/d/Y" datePickerType="simple">
-  <DatePickerInput
-    id="date-picker-default-id"
-    placeholder="mm/dd/yyyy"
-    labelText="Date Picker label"
-    type="text"
-  />
-</DatePicker>`}</ComponentDemo>
-
-## Single with calendar
-
 <ComponentDemo
```

Comment:
Date picker theme bugs
Themes not rendering correctly in the calendars
![image](https://user-images.githubusercontent.com/11670886/78406459-48973700-75c8-11ea-90d3-3a93489046d8.png)